src/main/java/org/folio/processing/mapping/mapper/reader/record/marc/MarcRecordReader.java

```diff
@@ -166,10 +174,22 @@ private Value readSingleField(MappingRule ruleExpression, boolean isRepeatableFi
     return MissingValue.getInstance();
   }
 
-  private void processMARCExpression(boolean arrayValue, boolean isRepeatableField, List<String> resultList, StringBuilder sb, String expressionPart, MappingRule ruleExpression) {
+  private void processMARCExpression(boolean arrayValue, boolean isRepeatableField, List<String> resultList, StringBuilder sb, StringBuilder multipleStringBuilder, String expressionPart, MappingRule ruleExpression) {
```

Comment:
please add a small comment describing what this method does.

Reply:
In case when token is marc process this token. This method add marc field/subfield value to result

Reply:
Thanks. I meant adding the comment to the code.